### PR TITLE
Patch 8

### PR DESCRIPTION
**Pull request by feliciien**
/claim #296

### CODE REVIEW

**Comment by ezzcodeezzlife** [comment]
there are two tutorials, and both do not meet the issue requirements, please make sure to cover them all 🚀 thanks you!

**Comment by feliciien** [comment]
@ezzcodeezzlife, this has been marked as “not planned.” Am I allowed to close my pull request, or is it necessary for me to continue working on it?

File: auto-gpt-tutorial-how-to-create-a-website-with-auto-gpt.mdx [hunk 1]
```diff
@@ -0,0 +1,132 @@
+---
+title: "Use AutoGPT to create  website  faster"
```
Comment (jmisilo) [inline auto-gpt-tutorial-how-to-create-a-website-with-auto-gpt.mdx:2]: please remove double spaces

Comment (ezzcodeezzlife) [reply auto-gpt-tutorial-how-to-create-a-website-with-auto-gpt.mdx:2]: > please remove double spaces

@feliciien

**Comment by ezzcodeezzlife** [comment]
> there are two tutorials, and both do not meet the issue requirements, please make sure to cover them all 🚀 thanks you!

still an issue!

**Comment by feliciien** [comment]
Ok

**Comment by ezzcodeezzlife** [comment]
@feliciien the content is too similar to your previous tutorials

**Review by ezzcodeezzlife — CHANGES_REQUESTED** [review]
⬆️

**Comment by feliciien** [comment]
Can you give me suggestions of content.?

**Comment by ezzcodeezzlife** [comment]
https://github.com/lablab-ai/community-content/issues/296 unfortunately closed

**Pull request closed without merging**